datalad_debian/new_reprepro_repository.py

```diff
@@ -133,11 +133,13 @@ def _setup_reprepro_ds(ds):
     repo.call_annex([
         'config', '--set', 'annex.addunlocked', 'include=db/*'])
 
+    main_readme = ds.pathobj / 'README'
+    main_readme.write_text(superdataset_readme)
     dist_readme = ds.pathobj / 'distributions' / 'README'
     dist_readme.parent.mkdir(parents=True, exist_ok=True)
     dist_readme.write_text(dist_subds_readme)
     yield from ds.save(
-        path=['conf', 'repo', dist_readme],
```

Comment:
This 'repo' used to be what 'www' is now. If things still work, this change is likely fine. Thx!